src/filechooser.c

```diff
@@ -439,7 +449,7 @@ handle_open (XdpImplFileChooser *object,
   if (!g_variant_lookup (arg_options, "accept_label", "&s", &accept_label))
     {
       if (strcmp (method_name, "OpenFile") == 0)
-        accept_label = _("_Open");
+        accept_label = multiple ? _("_Open") : _("_Select");
```

Comment:
@allanday what do you think the default button should say when selecting for a directory in the file chooser?